docs/components/Drawer/Web.stories.tsx

```diff
@@ -11,6 +11,14 @@ export default {
   component: Drawer,
   parameters: {
     viewMode: "story",
+    previewTabs: {
+      code: {
+        hidden: false,
+        extraImports: {
+          "@jobber/components/Drawer": ["DrawerGrid"],
```

Comment:
you technicall could add it like 

```suggestion
          "@jobber/components/Drawer": ["Drawer", "DrawerGrid"],
```

if you want it to be a clean import